src/neo4j_genai/schema.py

```diff
@@ -0,0 +1,143 @@
+#  Copyright (c) "Neo4j"
+#  Neo4j Sweden AB [https://neo4j.com]
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      https://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from typing import Any, Dict, List
```

Comment:
Maybe a detail but `Dict` and `List` from typing are deprecated. We can use the built-in types `dict` and `list` instead.

Reply:
[f87d866](https://github.com/neo4j/neo4j-genai-python/pull/32/commits/f87d866a82cbef61dfbb2c345d6acfc901f37d90)